Resources/Prototypes/_Gabystation/jukebox.yml

```
@@ -172,3 +182,9 @@
   name: Slipknot - Wait And Bleed (DoutorWhite mix)
   path:
     path: /Audio/_Gabystation/Lobby/wait_and_bleed.ogg
+
+- type: jukebox
+  id: Meu Piru
+  name: Allan Moreno - Quem vai querer o meu piru
+  path:
+    path: /Audio/_Gabystation/Lobby/QUEM_VAI_QUERER_O_MEU_PIRU.ogg
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)